README.md

```diff
@@ -1,6 +1,7 @@
 # tractolearn
 
 [![test, package](https://github.com/scil-vital/tractolearn/actions/workflows/test_package.yml/badge.svg?branch=main)](https://github.com/scil-vital/tractolearn/actions/workflows/test_package.yml?query=branch%3Amain)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7562790.svg)](https://doi.org/10.5281/zenodo.7562790)
```

Comment:
Can you add this Zenodo also https://zenodo.org/record/7562635#.Y-EXWNLMJH4

Reply:
Done in 063e1d3.